src/ca11y/index.js

```diff
@@ -105,7 +107,19 @@ class Ca11y {
       monthDisplay      : month + 1,
       fullYear          : date.getFullYear(),
       totalDays         : util.getDays(year, month),
-      firstWeekdayValue : util.getFirstDay(year, month)
+      firstWeekdayValue : util.getFirstDay(year, month),
+      parsed: {
+        d: day,
+        dd:  zeroPad(day),
+        ddd: this.props.days[date.getDay(day)].shortName,
+        dddd:  this.props.days[date.getDay(day)].fullName,
+        m: month + 1,
+        mm:  zeroPad(month + 1),
+        mmm: this.props.months[month].shortName,
+        mmmm:  this.props.months[month].fullName,
```

Comment:
Is there _ever_ a chance you'd hit a `Cannot ready property fullName of undefined`?